apps/web/src/scenes/UserGalleryPage/UserSharedInfo/UserSharedInfoList/SharedInfoListRow.tsx

```diff
@@ -17,6 +18,7 @@ type Props = {
 };
 
 export default function SharedInfoListRow({ title, subTitle, href, imageContent }: Props) {
+  const { hideModal } = useModalActions();
```

Comment:
this is a super duper nit, but could we move this down to line 37? generally like to define variables right next to where they're used to minimize mental overhead